### DISCUSSION TITLE
Error in pywa library for templates with image headers

### DISCUSSION THREAD

**Discussion by bcombes** [david-lev/pywa#14] [General]
I'm not sure how best to file a bug report, but I found an error in the pywa send_template library when using image headers. According to the facebook docs, the key for the image location should be `link` and not `url` which results in the error below;

> pywa.errors.WhatsAppError: WhatsAppError(message='(#100) Unexpected key "url" on param "template[\'components\'][0][\'parameters\'][0][\'image\']".', details=None, code=100)

I've been trying to submit a PR to fix this, but not sure how.

**Comment by david-lev** [comment]
Issue opened #15
